config/routes.rb

```diff
@@ -48,6 +48,9 @@
         end
       end
 
+      # record 2i outcomes
+      resources :two_eye_outcomes, only: %i[new create], controller: "editions/two_eye_outcomes", path_names: { new: "/new" }
```

Comment:
I wonder if `second_pair_of_eyes_outcomes` would be a better name here? I wonder if we should also be consistent and call the 2i-related states `awaiting_second_pair_of_eyes`. It's a bit more verbose, but keeps the consistency right across the codebase.